index.html

```diff
@@ -19,8 +19,6 @@
 </head>
 <body>
   <app-root></app-root>
-
-  <script type="module" src="../dist/demo/app-root.js">
-  </script>
+  <script type="module" src="./dist/src/app-root.js"></script>
```

Comment:
we had to move index.html up so that `yarn start` and `yarn prepare:ghpages` will both work targeting the same source. when pointing to a nested dir, Vite nests in a way that adds `/demo` in url. we'd rather have `https://internetarchive.github.io/<review-app-name>` and not `https://internetarchive.github.io/<review-app-name>/demo`